.circleci/continue_config.yml

```diff
@@ -1462,6 +1504,28 @@ jobs:
       - run:
           name: Run tests
           command: DD_TRACE_AGENT_TIMEOUT=1000 <<# parameters.disable_runner_distributed_tracing >> DD_DISTRIBUTED_TRACING=false <</ parameters.disable_runner_distributed_tracing >> DD_TRACE_TEST_SAPI=<< parameters.sapi >> make << parameters.make_target >> RUST_DEBUG_SYMBOLS=1 PHPUNIT_OPTS="--log-junit test-results/php-composer/results.xml"
+      - when:
+          # codecov uploader only on amd64
+          condition:
+            and:
+              - equal: [ true, << parameters.coverage >> ]
+              - matches:
+                  pattern: "^[^.]+$"
+                  value: << parameters.resource_class >>
+          steps:
+            - run:
+                name: Merge coverage reports
+                command: |
+                  make merge_coverage_reports
+            - run:
+                name: Install CodeCov Uploader Dependencies
+                command: |
+                  sudo apt update
+                  sudo apt install -y gpg
```

Comment:
Note: add gpg to base buster container.
Maybe just include it already in dockerfile.